src/config/getConfig.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Why this is necessary?

Reply:
If it isn't there then the setting `enum` is incorrectly marked as unused. 

Having the `enum` list there is useful when there is a missing item in one of the four config json files.